plugin/src/cli_utils/localpvlvm/mod.rs

```diff
@@ -0,0 +1,90 @@
+use crate::cli_utils::CliArgs;
+use clap::Parser;
+use plugin::ExecuteOperation;
+pub(crate) mod node;
+pub(crate) mod volume;
+use snafu::Snafu;
+
+/// LocalPV lvm operations.
+#[derive(Parser, Debug)]
+pub enum Operations {
+    /// Gets localpv-lvm resources.
+    #[clap(subcommand)]
+    Get(LvmGet),
+}
+
+#[async_trait::async_trait(?Send)]
+impl ExecuteOperation for Operations {
+    type Args = CliArgs;
+    type Error = Error;
+
+    async fn execute(&self, cli_args: &CliArgs) -> Result<(), Error> {
+        match self {
+            Operations::Get(lvmget) => {
+                lvmget.execute(cli_args).await?;
+            }
+        }
+        Ok(())
+    }
+}
+
+#[derive(clap::Subcommand, Debug)]
+pub enum LvmGet {
```

Comment:
missing doc comments throughout

Reply:
/// HERE
pub enum LvmGet {

Reply:
I commented that where it was referenced.

```
pub enum Operations {
    //// Gets localpv-lvm resources.
    #[clap(subcommand)]
    Get(LvmGet),
}
```

When the command executes its taking precedence here. But ill add in next PR for readability.